src/hotel_reservations/mlflow_processor.py

```
@@ -14,23 +23,32 @@
-        self.model_name = (
-            self.config["catalog_name"] + "." + self.config["schema_name"] + "." + "house_prices_model_basic"
-        )
+        self.model_name = model_name
+        self.full_model_path = self.config["catalog_name"] + "." + self.config["schema_name"] + "." + self.model_name
+        self.host = host
+        self.token = token
+
+        try:
+            workspace = WorkspaceClient()
+        except Exception:
+            workspace = WorkspaceClient(host=self.host, token=self.token)
```

Ruff

Local variable `workspace` is assigned to but never used

Remove assignment to unused variable `workspace`

(F841)